client/components/NetworkList.vue

```diff
@@ -123,6 +223,110 @@ export default {
 				order: channel.network.channels.map((c) => c.id),
 			});
 		},
+		toggleSearch(event) {
+			event.preventDefault();
+
+			if (this.$refs.searchInput === document.activeElement) {
+				this.closeSearch();
+				return;
+			}
+
+			this.activateSearch();
+		},
+		activateSearch() {
+			if (this.$refs.searchInput === document.activeElement) {
+				return;
+			}
+
+			this.sidebarWasOpen = this.$store.state.sidebarOpen;
+			this.$store.commit("sidebarOpen", true);
+			this.$nextTick(() => {
+				this.$refs.searchInput.focus();
```

Comment:
Add `scrollToActive` so it scrolls after clicking an entry in search.

Reply:
shouldn't that be done in `selectResult`?

Reply:
This should be fixed now.